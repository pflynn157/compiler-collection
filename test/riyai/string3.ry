
func main -> i32 is
    var s : string := "Hello!";
    
    var i : i32 := length(s);
    print(i);
    print(length(s));
    
    i := 0;
    while i < length(s) do
        var c : char := s[i];
        print(c);
        i := i + 1;
    end
    
    return 0;
end

