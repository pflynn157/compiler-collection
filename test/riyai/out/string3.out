6
6
H
e
l
l
o
!
